.github/scripts/js/e2e/slash_workflow_command.js

```diff
@@ -0,0 +1,93 @@
+const {abortFailedE2eCommand} = require("../constants");
+
+/**
+ * Build valid return object
+ * @param {string} flag for realease issue logic
+ * @param {object} workflow_id - workflow id
+ * @param {object} targetRef - workflow target ref
+ * @param {object} inputs - target workflow inputs
+ * @return {object}
+ */
+function buildReturn(flag, workflow_id, targetRef, inputs) {
```

Comment:
Hm.. this method uses only once. Why not return object in place, without 'build' method?

Reply:
yes. done